com.bitwarden.desktop.yaml

```
@@ -1,14 +1,17 @@
 app-id: com.bitwarden.desktop
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '24.08'
 sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
-base-version: '22.08'
+base-version: '24.08'
 command: bitwarden
+sdk-extensions:
+  # Node.js version: https://github.com/bitwarden/clients/blob/e2c25c6aad47d1e086ee8bb749e413cd93f89ca8/.nvmrc
+  - org.freedesktop.Sdk.Extension.node20
+  - org.freedesktop.Sdk.Extension.rust-stable
 finish-args:
   - --share=ipc
   - --share=network
-  - --socket=wayland
   - --socket=x11
```

Let's not remove wayland support in this day and age? :innocent: 

Instead, let's make x11 the fallback if we're not in a wayland session:
```suggestion
  - --socket=fallback-x11
```

---

Agreed. I'll note that the upstream PR (https://github.com/bitwarden/clients/pull/10359) to default to wayland is blocked at the moment because of an electron bug, so this will go via Xwayland either way.

---

Xwayland is not wayland. So in that case only `x11`

---

Ah, I had missed that; in this case the original change was correct, removing `--socket=wayland` and keeping `--socket=x11`; apologies for the bad suggestion 🙈